react/react1/week3/react1-week3/src/index.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import Main from "./Main";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+root.render(
+  <React.StrictMode>
+    <Main />
```

Comment:
I recommend you to use App as your main page because if you want to deploy this app, you may have a problem with that, also you can put the content of TodoListStyle.css on App.css.

Reply:
okay